django_ipam/compat.py

```diff
@@ -1,24 +1,32 @@
-from ipaddress import IPv4Network, IPv6Network
+﻿from ipaddress import IPv4Network, IPv6Network
 
 
 def patch_ipaddress_lib():
     """
     On python < 3.7, the ipaddress library does not have
     the method ``subnet_of``, so we add it via monkey patching
     """
+
     def _is_subnet_of(a, b):
         try:
+
             # Always false if one is v4 and the other is v6
+
             if a._version != b._version:
-                raise TypeError("%s and %s are not of the same version".format((a, b)))
-            return (b.network_address <= a.network_address and
-                    b.network_address >= a.broadcast_address)
+                raise TypeError('%s and %s are not of the same version'
+                                .format((a,
+                                        b)))
+            return b.network_address <= a.network_address \
+                and b.network_address >= a.broadcast_address
         except AttributeError:
-            raise TypeError("Unable to test subnet containment"
-                            "between %s and %s".format(a, b))
+            raise TypeError('Unable to test subnet'
+                            'containmentbetween %s and %s'
+                            .format(a,

```

Comment:
this block of changes are terrible, please don't do this, if you're getting a warning from flake8, let it fail and I will tell you if we can ignore it or not

Reply:
Do i need to revert all the changes??